### ISSUE TITLE
does not work on my mendeley

### ISSUE THREAD

**Issue by astroHaoPeng** [rdiaz02/Adios_Mendeley#3]
When I'm running this:
```R
bibfileFileFixed <- fixFileNames(bibfile2, tmpFilePaths)
```

I got these output from RStudio:
```R
 Copying file failed for  /E$\backslash$
 Copying file failed for  /E$\backslash$
 Copying file failed for  /E$\backslash$
 Copying file failed for  /E$\backslash$
 Copying file failed for  /E$\backslash$
 Copying file failed for  /E$\backslash$
Error in gsub(cc, "", ff, fixed = TRUE) : 
  input string 1 is invalid in this locale
In addition: There were 22 warnings (use warnings() to see them)
Called from: gsub(cc, "", ff, fixed = TRUE)
```

I think it might because the mendeley output the file direction like this:

```file = {:E$\backslash$:/BaiduYun/Mendeley/2009/2009{\_}Borne{\_}Scientific Data Mining in Astronomy.pdf:pdf}```

where it seems the `$\backslash$` cause some problem.

Can you fix that? I really want to move out of the mendeley as I have more and more papers to manage.

**Comment by rdiaz02** [comment]
I am sorry you are seeing these issues, but I think they are related to locale issues. As well, I think you are in Windows, and I would expect issues in Windows. Anyway, lets try this possible (ugly) fix:

1. Go to file `sqlite-bitex-functions.R`
2. Move to line 493, right after `oldpath <- gsub("\\_", "_", oldpath, fixed = TRUE)`
3. Insert this line
```
oldpath <- gsub(":E$\backslash$:", "E", oldpath, fixed = TRUE)
```
4. Run the whole thing again.

I think your files live in a hard drive named "E". What I am doing above is trying to obtain a valid path.

I am not 100% sure that will work. You might need to tweak the above expression a little.

**Comment by astroHaoPeng** [comment]
Yes, I have made there modifications you mentioned. After almost two days, it finally works. I'm really satisfied with the results and also your experience with Android tablets. However, since I have no time to clean up the modified codes and add some English comments, I forked your code and add some simple Chinese explanations to explain what happens to me when transferring on Windows platform.
Anyway, thanks for your sharing.
